### ISSUE TITLE
Bug in RPC 153

### ISSUE THREAD

**Issue by Zelyutin** [BrunoBM16/samp-packet-list#14]
Now in SetPlayerSkin - ID: 153 we have following parameters: UINT16 wPlayerID, UINT32 dSkinID
But there's 2 more bytes between playerID and skinID actually.

As I saw in SAMP 0.3DL Compability solution, playerId is processing like UINT32 there, so those 2 bytes are added to player ID:
https://github.com/AGraber/samp-dl-compat/blob/bc4e2676b95a5f4dc62e99e63dc37b08718dfea9/dl-compat.inc#L113

I'd rather say that there's UINT16 UnknownParameter after PlayerID to let PlayerID be UINT16 as it is in the other RPCs.

**Comment by BrunoBM16** [comment]
I have updated the 0.3.DL page.

Thanks.

**Comment by Zelyutin** [comment]
It's not about 0.3 DL actually.
You've added at 0.3 DL page this:
```
SetPlayerSkin - ID: 153
Parameters: UINT32 dwPlayerID, UINT32 dBaseSkinID, UINT32 dCustomSkinID
```
So, there's three UINT32, but it should be 16+32+32 following that code above.

While on general page you have:
```
SetPlayerSkin - ID: 153
Parameters: UINT16 wPlayerID, UINT32 dSkinID
```
So, there's no third parameter dCustomSkinID, that's right as there no custom skins in 0.3.7, but first parameter should be UINT32. Why? I don't actually now, but I've found on practice that whole structure is 8 bytes long. And in the code above those 2 bytes are placed in first parameter in the code above making it UINT32 instead of UINT16.

So, briefly, current structure won't give correct results in 0.3.7. I've tested it. 32 + 32 is okay.
Seems like Kalcor fixed it in 0.3 DL and made playerID be uint16 as it was before.
